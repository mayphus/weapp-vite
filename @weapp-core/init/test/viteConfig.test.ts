import { getDefaultViteConfig } from '@/viteConfig'

describe('viteConfig', () => {
  it('getDefaultViteConfig', () => {
    expect(getDefaultViteConfig()).toMatchSnapshot()
  })
})
