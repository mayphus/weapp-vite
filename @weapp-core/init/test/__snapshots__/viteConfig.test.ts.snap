// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`viteConfig > getDefaultViteConfig 1`] = `
"import { defineConfig } from 'weapp-vite/config'

export default defineConfig({
  weapp: {
    // weapp-vite options
  },
})
"
`;
